trec_covid_semantic_search/workload.json

```diff
@@ -0,0 +1,30 @@
+{% import "benchmark.helpers" as benchmark with context %}
+
+{
+  "version": 2,
+  "description": "Benchmark performance of semantic search queries based on dataset of global daily weather measurements from NOAA",
+  "indices": [
+    {
+      "name": "trec-covid",
+      "body": "index.json"
+    }
+  ],
+  "corpora": [
+    {
+      "name": "trec-covid",
+      "base-url": "https://github.com/martin-gaievski/neural-search/releases/download/trec_covid_dataset_1M_v1",
```

Comment:
i believe this is temporary. You must be using generic repository to fetch corpus